app/routes/shop.py

```diff
@@ -114,4 +118,34 @@ def upload_stamp_image():
   response.raise_for_status()
   data = response.json()
 
-  return { 'data':  data }
\ No newline at end of file
+  dive_shop = DiveShop.query.get_or_404(id)
+  setattr(dive_shop, 'stamp_uri', data.get('uri'))
+  db.session.commit()
+
+  return 'ok'
```

Comment:
instead of returning 'ok' can we return either the updated dive shop object, or at least `{ 'msg': 'dive shop successfully updated' }`, to match our existing pattern for return messages from the API